src/blocks/Login/Login.spec.tsx

```diff
@@ -0,0 +1,28 @@
+describe('Login', () => {
+    it(`Should render the form`, () => {
+        cy.visit(`${Cypress.env('baseUrl')}/iframe.html?id=blocks-login--base`)
+        cy.get(`form`).should('exist')
+    })
+    it(`Should render username field`, () => {
+        cy.visit(`${Cypress.env('baseUrl')}/iframe.html?id=blocks-login--base`)
+        cy.get(`#user_login`).should('exist')
+    })
+
+    it(`Should render password field`, () => {
+        cy.visit(`${Cypress.env('baseUrl')}/iframe.html?id=blocks-login--base`)
+        cy.get(`#user_pass`).should('exist')
+    })
+
+    it(`Should have submit`, () => {
+        cy.visit(`${Cypress.env('baseUrl')}/iframe.html?id=blocks-login--base`)
+        cy.get('form').submit().should('exist')
+    })
+
+    it(`Submit Form: Should test form submition`, () => {
+        cy.visit(`${Cypress.env('baseUrl')}/iframe.html?id=blocks-login--base`)
+        cy.get('#login_submit').click()
+        cy.on('window:alert', (txt) => {
+            expect(txt).to.contains('Login')
+        })
+    })
+})
```

Comment:
Great job!
Quick add-ons:
Could you, please, add tests to be sure the block is rendering:
- b-login
- u-block
- title (we are going to need a new story on the stories without a title)

We are not testing the form's classes/elements because we are going to replace it with rds forms in the future :)

Reply:
Title is now optional, more tests were added.

Reply:
Great job!